packages/docs/docs/tutorial-basics/basic-fhir-search.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 4
+---
+
+# Basic Search in FHIR on Medplum
+
+This is an introductory post to searching in FHIR on Medplum.  It will introduce the basic concepts and provide some sample code on how to execute basic searches.  We won't cover all the FHIR search scenarios, for in depth reference material on FHIR search, [this document](https://www.hl7.org/fhir/search.html) is a good place to start.
+
+There are two high level concepts that is is useful to understand when learning about Search in FHIR:
+
+* FHIR Objects - for example [Patient](https://www.hl7.org/fhir/patient.html)
+* Search Parameters - these are searchable fields on a given FHIR object
+
+Using these two basic concepts, let's walk through an example where you want to query all the Patient objects for those who have a name like "Alex".
+
+In the example below `Patient` is the FHIR object and `name` is the search parameter.  As the `Patient` class has many fields, so specifying which fields you want to query in the API, in the example you can see this as `_fields`.
+
+To execute this search as via API, here's how you would do it using curl.  In this example, you are limiting it the results to 20.
+
+```curl
+curl 'https://api.medplum.com/fhir/R4/Patient?_count=20&_fields=id,_lastUpdated,name,birthDate,gender&_sort=-_lastUpdated&name=Alex' \
+  -H 'authorization: Bearer $ACCESS_TOKEN' \
+  -H 'content-type: application/fhir+json' \
```

Comment:
You shouldn't need `Content-Type` header for  `GET` request, only `POST` AND `PUT`